Makefile.am

```diff
@@ -2431,6 +2432,20 @@ sbus_internal_tests_LDADD = \
     libsss_debug.la \
     libsss_test_common.la
 
+config_check_tests_SOURCES = \
+    src/tests/cmocka/config_check.c \
```

Comment:
All test files in this directory have prefix or suffix test. Rest of files are auxiliary files.
It would be good to follow this convention.